.gitlab-ci.yml

```diff
@@ -2948,11 +2948,6 @@ twistlock_scan-7:
       If (`$lastExitCode -ne "0") { throw "Previous command returned `$lastExitCode" }
       Remove-Item `$PWD\docker.key
       "@ | out-file ci-scripts/docker-publish.ps1
-  after_script:
-    - $ErrorActionPreference = "Stop"
```

Comment:
Shouldn't we add this to the other jobs? Just in case we add further lines to the jobs in the future

Reply:
👍 I assumed it was there already but it wasn't. Added that together with some missing `$lastExitCode` checks